crates/wasmtime/src/config.rs

```diff
@@ -17,6 +17,7 @@ use wasmtime_jit::profiling::{self, ProfilingAgent};
 use wasmtime_runtime::{InstanceAllocator, OnDemandInstanceAllocator, RuntimeMemoryCreator};
 
 pub use wasmtime_environ::CacheStore;
+pub use wasmtime_runtime::AutoEnabled;
```

Comment:
We might want to name this something a little more specific to MPK. `MpkEnabled`?

Alternatively, I can imagine other scenarios where we will want tri-state, should-we-enable enums like this and we could keep this name but then make its documentation not mention MPK.

Reply:
Renamed to `MpkEnabled`.